cibuildwheel/windows.py

```diff
@@ -102,7 +115,17 @@ def shell(args, env=None, cwd=None):
     for config in python_configurations:
         # install Python
         config_python_path = get_python_path(config)
-        simple_shell([nuget, "install"] + get_nuget_args(config))
+        if config.identifier.startswith('cp'):
+            simple_shell([nuget, "install"] + get_nuget_args(config))
+        elif config.identifier.startswith('pp') and not os.path.exists(config_python_path):
+            pypy_zip = os.path.join("C:\\cibw", config.url.rsplit('/', 1)[-1])
+            download(config.url, pypy_zip)
+            # Extract to the parent of config_python_path because the zip file still contains a directory
+            extract_zip(pypy_zip, os.path.dirname(config_python_path))
+            pypy_exe = 'pypy.exe' if config.version[0] == '2' else 'pypy3.exe'
+            simple_shell(['mklink', os.path.join(config_python_path, 'python.exe'), os.path.join(config_python_path, pypy_exe)])
+            simple_shell(['mklink', '/d', os.path.join(config_python_path, 'Scripts'), os.path.join(config_python_path, 'bin')])
+            simple_shell(['chcp', '437'])  # Workaround for https://bitbucket.org/pypy/pypy/issues/3081/
```

Comment:
To be fixed in PyPy 7.2.1 or 7.3.0, whichever release is next?

Reply:
yes, that is already fixed on py3.6 HEAD but I think this is needed for cpython2 and pypy2

Reply:
Hmmm, I don't think we need this for CPython2, but I can't remember if the tests only failed on PyPy 3 or also on PyPy 2. But we'll notice, when we update the PR; thanks! :-)